corehq/form_processor/management/commands/fix_edit_forms_attachments.py

```diff
@@ -0,0 +1,156 @@
+        if index > 100:
+            if form_.domain not in IGNORE_DOMAIN:
+                print("too high index for form %s. Hence kept just till 10" % orig_form_id)
+            return [form_]
+        return get_sql_previous_versions(form_.deprecated_form_id, form_accessor, index, orig_form_id) + [form_]
+    else:
+        return [form_]
+
+
+class Command(BaseCommand):
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.forms = {}
+        self.search_after_feature_release = False
+
+    def add_arguments(self, parser):
+        parser.add_argument('--source', help='sql/couch')
+        parser.add_argument('--update', action='store_true', help='actually update')
+        parser.add_argument('--inspect', action='store_true', help='just write findings to file')
+        parser.add_argument('--after', action='store_true',
+                            help='search for edits after edit form feature release')
+
+    def _find_sql_forms_with_missing_attachments(self):
+        """
+        find all forms that are a result of a form edition
+        find the deprecated form and then compare attachments between them
+        :return:
+        """
+        add_forms_with_attachments = defaultdict(list)
+        form_accessor = FormAccessorSQL()
+        for db_alias in get_db_aliases_for_partitioned_query():
+            print('scanning %s' % db_alias)
+            edit_form_ids = (XFormOperationSQL.objects.using(db_alias)
+                             .filter(operation='edit')
+                             .values_list('form_id', flat=True).order_by('-date').distinct())
+            for edit_form_id in with_progress_bar(edit_form_ids):
+                if edit_form_id in self.forms:
+                    # this form was already covered by some other form in its hierarchy
+                    continue
+                edit_chain = get_sql_previous_versions(edit_form_id, form_accessor)
+                original_form = edit_chain[0]
+                original_attachments = list(FormAccessorSQL.get_attachments(original_form.form_id))
+                for edited_form in edit_chain[1:]:
+                    attachments = list(FormAccessorSQL.get_attachments(edited_form.form_id))
+                    attachment_names = [a.name for a in attachments]
+                    for attachment in original_attachments:
+                        if attachment.name != 'form.xml' and attachment.name not in attachment_names:
+                            assert attachment.parent_id == original_form.form_id
+                            self.forms[edited_form.form_id] = edited_form
+                            for attachment_to_be_added in add_forms_with_attachments[edited_form.form_id]:
+                                if attachment_to_be_added.key == attachment.key:
+                                    # this attachment add request is already present
+                                    continue
```

Comment:
What is this `continue` supposed to do? I think it continues with the next iteration on line 93 (3 lines above), which is the same thing that happens when `attachment_to_be_added.key != attachment.key`. This entire `for`-loop appears to do nothing.